### ISSUE TITLE
Discriminator loss going to zero while training

### ISSUE THREAD

**Issue by sumandroid12** [iwtw/pytorch-TP-GAN#6]
Hello can you explain how you trained your network. I have observed that D_loss goes to zero rapidly and all my gradients "vanish" as i start training.
